share/spice/isslocation/isslocation.js

```diff
@@ -0,0 +1,38 @@
+(function(env) {
+    "use strict";
+    env.ddg_spice_isslocation = function(api_result) {
+        
+       if(!api_result || api_result !== "success") {
```

Comment:
Just the latter one. `!api_result` says, "check if api_result is not null or undefined" and `api_result.message !== "success"` says that the `message` property of `api_result` should equal to "success" to continue.

Looking at http://your-codio-instance:5000/js/spice/isslocation/ should make it clearer. :)